websites/T/TOD/presence.ts

```diff
@@ -0,0 +1,296 @@
+class TOD extends Presence {
+	constructor(presenceOptions: PresenceOptions) {
+		super(presenceOptions);
+	}
+
+	getVideo() {
+		return document.querySelector<HTMLMediaElement>("video");
+	}
+
+	getVideoType() {
+		if (
+			this.getVideoTitle().match(/S\d+\s*\|\s*E\d+/i) ||
+			this.getVideoTitle().match(/EP\s*\d+/i)
+		)
+			return "show";
+		else return "movie";
+	}
+
+	getTitle(eyebrow = false) {
+		if (this.isWatching() && eyebrow) return this.getVideoTitle();
+
+		const title = document.querySelector(
+			"div.bein-dh1-hero__head-container > h1.bein-dh1-hero__title.heading-shadow"
+		)?.textContent;
+
+		return (
+			title ??
+			this.getVideoTitle().replace(/\d+\s*-\s*\d+/ as RegExp, "vs") ??
```

Comment:
Looks like an unnecessary type casting.
```suggestion
			this.getVideoTitle().replace(/\d+\s*-\s*\d+/, "vs") ??
```

Also, `getVideoTitle` returns `string | undefined`. Currently, the third part would not be reached, but an error would be thrown in the case of `undefined`. It should be null checked or the third condition should be removed.
```suggestion
			this.getVideoTitle()?.replace(/\d+\s*-\s*\d+/, "vs") ??
```